### ISSUE TITLE
application restart not working as well

### ISSUE THREAD

**Issue by Emy87aa** [MoathOthman/MOLH#46]
![WhatsApp Image 2019-12-04 at 2 17 24 PM](https://user-images.githubusercontent.com/1811875/70142947-140b8100-16a3-11ea-92f0-1815fd8b693b.jpeg)

When restarting the app this happens, nil in rootViewController

**Comment by MoathOthman** [comment]
Hey eman , can you provide more details about the environment . 
What did you do ?

**Comment by Emy87aa** [comment]
I edited the issue and added screen Shot

**Comment by supergenedy** [comment]
@Emy87aa 
I made a solution for this issue till MoathOthman updates the lib
you can add this func in your AppDelegate it gets the window from SceneDelegate

```
 @available(iOS 13.0, *)
    func swichRoot(){
        // switch root view controllers
        
        let storyboard = UIStoryboard.init(name: "Main", bundle: nil)
        let nav = storyboard.instantiateViewController(withIdentifier: "HomeView")
        
        let scene = UIApplication.shared.connectedScenes.first
        if let sd : SceneDelegate = (scene?.delegate as? SceneDelegate) {
            sd.window!.rootViewController = nav
        }
        
    }
```
You can check if ios > 13 then call this func from the AppDelegate

```
        if #available(iOS 13.0, *) {
                let delegate = UIApplication.shared.delegate as? AppDelegate
                delegate!.swichRoot()
        } else {
               // Fallback on earlier versions
               MOLH.reset()
        }
```
And also check if ios > 13 in the reset func in the AppDelegate to call swichRoot func

**Comment by MoathOthman** [comment]
@supergenedy thanx for the solution .

**Comment by imuhammadjabbar** [comment]
> @supergenedy thanx for the solution .

@MoathOthman it does solve the animation crash issue but the language is not getting translated.

**Comment by supergenedy** [comment]
@imuhammadjabbar did you call MOLH.setLanguageTo("YOUR_LANGUAGE") ?

**Comment by imuhammadjabbar** [comment]
> @imuhammadjabbar did you call `MOLH.setLanguageTo("YOUR_LANGUAGE")` ?

Yes, I did. ( MOLH.setLanguageTo(currentLanguage == "en" ? "ar" : "en") )
I set my default Language to Ar.

Seems like it is changing now. But My Side Menu is not working. or some images.
How can I change images manually(There were changing automatically with Changing System LAnguage)

And the side menu is not working as well.

Any help would be much appreciated. Thanks

**Comment by MoathOthman** [comment]
@imuhammadjabbar there is a function for images `image.flipIfNeeded()`

**Comment by abdallahnh** [comment]
The crash is happening because we are forced unwrapping the window

**Comment by MoathOthman** [comment]
fixed in 1.3
